srtcore/fec.cpp

```diff
@@ -468,7 +470,7 @@ void FECFilterBuiltin::feedSource(CPacket& packet)
             return;
         }
 
-        SRT_ASSERT(vert_off >= 0);
+        // [[assert vert_off >= 0]]; // this condition branch
```

Comment:
What's the point of changing to a comment?

Reply:
To make things clear that this condition is satisfied here, should you have a thought at some next time that maybe an assertion could be a good idea. Some of these conditions are not always obvious, especially if you are deep in the long branch.

Reply:
That's the goal of the assertion, not a comment., to assert a condition is satisfied.

Reply:
Assertion is for a condition that should be satisfied despite that this should result from some other conditions. Not for situations like:
```
if ( a > 0)
{
     ASSERT(a > 0);
...
```

Reply:
Ah, ok, missed that!